scripts/test-harness-contracts.ps1

```
@@ -400,17 +415,28 @@ try {
+    Check-Match '.claude/skills/pelizzai-execute/templates/state.md' 'LOCAL per dev' 'state.md template declares the cursor local per dev (issue #43)'
+    Check-Match '.claude/skills/pelizzai-starting-branch/SKILL.md' 'fresh worktree starts WITHOUT the cursor' 'starting-branch: a new worktree does not carry the ignored cursor (issue #43)'
+    Check-Match '.claude/skills/pelizzai-recovery/SKILL.md' 'cursor-only recovery needs no commit' 'recovery: cursor-only recovery has no commit (issue #43)'
+    Check-Match '.claude/skills/pelizzai-execute/references/task-cycle.md' 'NEVER staged[\s\S]{0,220}no cursor in any commit' 'task-cycle §7: the cursor is never staged, in any mode (issue #43)'
+    Check-NotMatch '.claude/skills/pelizzai-execute/references/task-cycle.md' 'include it in the stage|carries the cursor' 'task-cycle: the old stage-the-cursor doctrine is gone (issue #43)'
+    Check-NotMatch '.claude/skills/pelizzai-execute/SKILL.md' 'cursor included' 'execute: the flow diagram no longer puts the cursor inside the commit (issue #43)'
 
     # -- The cursor deflates at CLOSEOUT (the delivered seal), not at the next opening --
     Check-Match '.claude/skills/pelizzai-execute/SKILL.md' 'Migration at the .delivered' 'execution-plans: the history/ migration happens at the delivered seal'
     Check-Match '.claude/skills/pelizzai-finish/SKILL.md' 'Migrate the intact block and deflate the cursor' 'finish-task runs the migration when sealing delivered'
-    Check-Match '.claude/skills/pelizzai-finish/SKILL.md' 'git add -- pelizzai/data/state\.md pelizzai/data/history/' 'finish-task stages state + history in the same closure'
+    Check-Match '.claude/skills/pelizzai-finish/SKILL.md' 'git add -- pelizzai/data/history/' 'finish-task stages ONLY the migrated history file in the closure (issue #43)'
```

PSScriptAnalyzer (1.25.0)

[info] Cmdlet 'Check-Match' has positional parameter. Please use named parameters instead of positional parameters when calling a command.

(PSAvoidUsingPositionalParameters)